free-1.md

```diff
@@ -57,12 +155,31 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Honestly, a lot of this work would focus on the "systems" aspects because 
+dealing with audio is hard. And I'm not sure how this DSL would be implemented. 
+But breaking ornaments into small patterns is a part of language design. 
+Moreover, the language design involves connecting these pieces not only in 
+sequence, but also by tempo and other embellishments.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is an ambitious idea because I'm trying to encode music that 
+currently has ambiguous encodings. I also have little direction into my 
+project, so it feels big.
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This might be a good idea because I am trying to make different musical 
+interpretations clearer. In addition, this could potentially be used by people 
+in the music domain, since ornaments are difficult for computers to express.
+
+One drawback is that I can't see how this DSL would grow. There is only a 
+finite number of ornaments and a finite number of interpretations, even if this 
+number is large. And I want to limit this interpretation project to just 
+musical ornaments. I'm not saying DSLs must grow, but at the same time, aren't 
+well-designed languages able to grow?
```

Comment:
They should grow in the sense that people may use it for things or in ways that you hadn't imagined, but still within the domain. Thus a language should be able to grow within the domain to accommodate that, but if your language has truly grown to the point that it encompasses the entire domain (which I think is unlikely), then there isn't a reason for it to grow (in scope at least).